src/main/webapp/search-interview.js

```diff
@@ -21,16 +21,30 @@ function onSearchInterviewLoad() {
 // Queries Datastore for available interview times and renders them on the
 // page.
 function loadInterviews() {
-  const searchResultsDiv = document.getElementById("search-results");
-  searchResultsDiv.removeAttribute("hidden");
-  fetch(`/load-interviews?timeZoneOffset=${browserTimezoneOffset()}&position=${selectedPosition()}`)
-  .then(response => response.text())
-  .then(interviewTimes => {
-    interviewTimesDiv().innerHTML = interviewTimes;
-  });
+  const searchResultsDiv = document.getElementById('search-results');
+  searchResultsDiv.removeAttribute('hidden');
+  const role = selectedRole();
+  if (role === 'Interviewee') {
+    fetch(`/load-interviews?timeZoneOffset=${browserTimezoneOffset()}&position=${selectedEnumPosition()}`)
```

Comment:
Nit: these two cases are basically the same. l'd say define a string that's either 'load-interviews' or 'shadow-load-interviews' and put that into the fetch URL

Reply:
Removed repeats.